webauthn/helpers/base64url_bytes.py

```diff
@@ -0,0 +1,40 @@
+from base64 import urlsafe_b64decode
+from typing_extensions import Annotated
+
+from pydantic import (
+    AfterValidator,
+    BeforeValidator,
+    FieldValidationInfo,
+    PlainSerializer,
+    WithJsonSchema,
+)
+from pydantic_core import PydanticCustomError
+
+from webauthn.helpers import bytes_to_base64url
+
+
+def validate_base64url(value: bytes, info: FieldValidationInfo) -> bytes:
+
+    if info.mode == "json":
+        try:
+            return urlsafe_b64decode(value + b"====")
+        except ValueError as e:
+            raise PydanticCustomError(
+                "base64_decode", "Base64 decoding error: '{error}'", {"error": str(e)}
+            ) from e
+
+    return value
+
+
+Base64URLBytes = Annotated[
```

Comment:
`Base64URLBytes` seems confusing to me because at first glance "is it string bytes? Or maybe just a byte array?" 

The fact that this gets encoded and decoded between base64url and bytes feels lost in the name. I keep trying to think of a better name for this, though, and don't have any suggestions yet. 🤔